### ISSUE TITLE
Benchmarks in docs

### ISSUE THREAD

**Issue by adam-coster** [JujuAdams/SNAP#6]
Since there are built-in methods doing some of the stuff that SNAP does, and since in general it's useful to know what the performance of tools is going to be, it would be great to have some benchmarks in the docs. In particular head-to-heads against built-ins, where available.

**Comment by JujuAdams** [comment]
It's going to be slower than native in basically all situations, and likely substantially so.

**Comment by adam-coster** [comment]
Fair enough! Mostly I'm curious if it's slower enough to *matter* in most contexts.
